eng/pipelines/runtimelab.yml

```
@@ -49,139 +49,71 @@ extends:
-          - wasi_wasm_win
-          jobParameters:
-            timeoutInMinutes: 300
-            isOfficialBuild: ${{ variables.isOfficialBuild }}
-            testGroup: innerloop
-            postBuildSteps:
-              - template: /eng/pipelines/runtimelab/runtimelab-post-build-steps.yml
-                parameters:
-                  isOfficialBuild: ${{ variables.isOfficialBuild }}
-                  uploadLibrariesTests: ${{ eq(variables.isOfficialBuild, false) }}
-                  uploadIntermediateArtifacts: ${{ variables.isOfficialBuild }}
-                  librariesConfiguration: Release
-            ${{ if eq(variables.isOfficialBuild, false) }}:
-              buildArgs: -s clr.aot+libs+nativeaot.packages -c $(_BuildConfig) /p:ArchiveTests=true
-            ${{ if eq(variables.isOfficialBuild, true) }}:
-              buildArgs: -s clr.aot+libs+nativeaot.packages -c $(_BuildConfig)
-=======
         - ${{ if ne(variables.isOfficialBuild, true) }}:
           #
-          # Build the whole product with Checked CoreCLR and run runtime tests
+          # Build the whole product with Debug CoreCLR and run runtime tests
           #
           - template: /eng/pipelines/common/platform-matrix.yml
             parameters:
               jobTemplate: /eng/pipelines/common/global-build-job.yml
               helixQueuesTemplate: /eng/pipelines/coreclr/templates/helix-queues-setup.yml
-              buildConfig: checked
+              buildConfig: debug
               platforms:
-              - linux_x64
+#              - linux_x64
+              - osx_x64
               - windows_x64
+              - Browser_wasm_win
+              - wasi_wasm_win
               jobParameters:
-                timeoutInMinutes: 200
-                buildArgs: -s clr+libs+hosts+packs -c debug -rc $(_BuildConfig)
+                timeoutInMinutes: 300
+                buildArgs: -s clr.aot+libs+nativeaot.packages -c debug -rc $(_BuildConfig)
                 postBuildSteps:
-                  - template: /eng/pipelines/common/templates/runtimes/build-runtime-tests-and-send-to-helix.yml
+                  - template: /eng/pipelines/runtimelab/runtimelab-post-build-steps.yml
                     parameters:
-                      creator: dotnet-bot
-                      testRunNamePrefixSuffix: CoreCLR_$(_BuildConfig)
+                      librariesConfiguration: Debug
 
           #
-          # Build the whole product with Release CoreCLR and run libraries tests
+          # Build with Debug libraries and Checked runtime
           #
           - template: /eng/pipelines/common/platform-matrix.yml
             parameters:
               jobTemplate: /eng/pipelines/common/global-build-job.yml
-              helixQueuesTemplate: /eng/pipelines/libraries/helix-queues-setup.yml
-              buildConfig: release
+              helixQueuesTemplate: /eng/pipelines/coreclr/templates/helix-queues-setup.yml
+              buildConfig: Checked
```

```suggestion
              buildConfig: checked
```
Since we're using lower-case configs now...